.github/workflows/dactyl.yaml

```diff
@@ -0,0 +1,78 @@
+name: "Dactyl build only on PR"
+
+on:
+  pull_request:
+    types: [opened, edited, synchronize]
+
+jobs:
+  build:
+    name: Build Dactyl Site
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Python Setup
+        uses: actions/setup-python@v1
+        with:
+          python-version: "3.7"
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install Jinja2==2.11.3
+          pip install dactyl lxml
+          pip install --upgrade dactyl
+      # - name: Run Dactyl Link Checker
+      #   run: |
+      #     dactyl_link_checker -p /blog/
+      #
```

Comment:
Let's remove the legacy comments (though we totally should expand this to run the link checker and report broken links at some point)

```suggestion
```